doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -5,21 +5,91 @@
+|Node|Nodes are the worker machines (physical or virtual) that run your containerised applications and other workloads. Each node is managed from the master, which receives updates on each node's self-reported status. A node runs the services necessary to support the Docker containers that make up your cluster's workloads. These include the Docker runtime and the Kubernetes node agent (kubelet) which communicates with the master and is responsible for starting and running Docker containers scheduled on that node.|[Link](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture)|
+|PaaS|The capability provided to the consumer is to deploy onto the cloud infrastructure consumer-created or acquired applications created using programming languages, libraries, services, and tools supported by the provider. The consumer does not manage or control the underlying cloud infrastructure including network, servers, operating systems, or storage, but has control over the deployed applications and possibly configuration settings for the application-hosting environment.|[NIST SP 800-145, The NIST Definition of Cloud Computing](https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-145.pdf)|
+|CaaS|Container-as-a-Service, a foundational application PaaS capability, in this case delivered as a Kubernetes cluster and other technologies such as CNI, Ingress, Service Mesh, LMA, etc. CaaS sits above VIM/NFVI and performs some application LCM based on instructions received from VNFM.|[Foundational PaaS Definition](http://www.theenterprisearchitect.eu/blog/categorize-compare-cloud-vendors/#row5column3)|
+|CaaS Manager|A management plane function that manages the lifecycle (instantiation, scaling, healing, etc.) of one or more Kubernetes clusters, including communication with VIM for master/node lifecycle management.|-|
+|CNF|A Cloud Native Virtual Network Function - or CNF for short - is an implementation of a Virtual Network Function (as defined by ETSI GS NFV 003 V1.4.1 (2018-08) that adheres to the CNCF Cloud Native Definition.|CNCF TUG (link tbc)|
+|Service|A Service in Kubernetes is an abstraction which defines a logical set of Pods and a policy by which to access them. Services enable a loose coupling between dependent Pods.|[Link](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/)|
+|Service Mesh|A service mesh is a dedicated infrastructure layer for handling service-to-service communication. It’s responsible for the reliable delivery of requests through the complex topology of services that comprise a modern, cloud native application. In practice, the service mesh is typically implemented as an array of lightweight network proxies that are deployed alongside application code, without the application needing to be aware.|[Link](https://buoyant.io/2017/04/25/whats-a-service-mesh-and-why-do-i-need-one/)|
```

Comment:
Let's keep terminology sparse, adding terms only when needed. Service Mesh (esp. in the k8s sense) is not an obvious one.

Reply:
May I suggest that we add the following note to "Service" -- "Service is a unit of consumption as it makes applications externally accessible."

Reply:
> May I suggest that we add the following note to "Service" -- "Service is a unit of consumption as it makes applications externally accessible."

That's not always true though (if you mean external to k8s) - e.g. ClusterIP is a type of Service that only makes applications accessible within a cluster.

Reply:
also headless Services don't even have ClusterIPs :)
in which case a Service is for service discovery, but not for traffic routing / loadbalancing